conf/messages

```diff
@@ -139,6 +139,13 @@ pages.voluntary.registration.para2                            = You can still re
 pages.voluntary.registration.reason.panels.neither.para       = If none of these apply you won’t be able to register voluntarily for VAT.
 validation.voluntary.registration.missing                     = Tell us if you want to register voluntarily for VAT
 
+## Chose No To Voluntary Registration Page
+pages.voluntary.registration.no.title                         = You’ve chosen not to register the company for VAT
```

Comment:
can you change the quote to this these, to keep the messages file consistent     ''

and the line below